multimodal/tests/unittests/test_object_detection.py

```diff
@@ -0,0 +1,40 @@
+import numpy as np
+import pytest
+import requests
+from PIL import Image
+
+from autogluon.multimodal import MultiModalPredictor
+
+
+def download_sample_images():
+    url = "https://raw.githubusercontent.com/open-mmlab/mmdetection/master/demo/demo.jpg"
+    image = Image.open(requests.get(url, stream=True).raw)
+    mmdet_image_name = "demo.jpg"
+    image.save(mmdet_image_name)
+
+    return mmdet_image_name
+
+
+@pytest.mark.parametrize(
+    "checkpoint_name",
+    [
+        "faster_rcnn_r50_fpn_2x_coco",
+        "yolov3_mobilenetv2_320_300e_coco",
+        "cascade_rcnn_x101_64x4d_fpn_20e_coco",
+        "detr_r50_8x2_150e_coco",
+    ],
+)
+def test_mmdet_object_detection_inference(checkpoint_name):
+    mmdet_image_name = download_sample_images()
+
+    predictor = MultiModalPredictor(
+        hyperparameters={
+            "model.names": ["mmdet_image"],
```

Comment:
Is `"model.names": ["mmdet_image"]` necessary here?

Reply:
I just followed the TIMM example [here](https://github.com/awslabs/autogluon/blob/master/multimodal/tests/unittests/test_zero_shot.py#L121). For now, it is not necessary, but future it might. We may include other detectors that are not in MMdetection. e.g., YOLOv7 and detectors trained on our BigDetection. I'm fine with either keeping it or removing it. What do you think?

Reply:
If we want to simplify user input, we can infer the model.names = xxx if user has input for model.xxx.checkpoint_name.

Reply:
As mentioned by @FANGAreNotGnu  I have removed this line.